src/relay/pushservice/pushservice.py

```diff
@@ -152,6 +153,9 @@ def _build_data_prop(event: Event):
     elif isinstance(event, TrustlineRequestEvent):
         if event.direction == "received":
             data = _get_data_prop_dict(event)
+    elif isinstance(event, TrustlineRequestCancelEvent):
+        if event.direction == "received":
+            data = _get_data_prop_dict(event)
```

Comment:
Could refactor:
```
elif isinstance(event, x) or isinstance(event, y) or isinstance(event, z):
    if event.direction == "received":
        data = _get_data_prop_dict(event)
        data = _get_data_prop_dict(event)
```

Reply:
Yes will do. Just saw that you can even do
```py
elif isinstance(event, (x, y ,z))
```